hyo2/abc2/lib/gdal_aux.py

```diff
@@ -217,6 +197,14 @@ def check_proj4_data(cls, verbose: bool = False) -> None:
                 logger.debug("already set PROJ_LIB = %s" % os.environ['PROJ_LIB'])
             return
 
+        try:
+            proj_path = pyproj.datadir.get_data_dir()
+            if verbose:
+                logger.debug("pyproj data dir = %s" % proj_path)
+            return
+        except Exception as e:
+            logger.warning("%s" % e)
+
```

Comment:
Integrated in https://github.com/hydroffice/hyo2_abc2/commit/80762d63be6856e39e153ff3814e32f39d8ee1dc